src/framework/shortcuts/view/shortcutsmodel.cpp

```diff
@@ -79,7 +79,8 @@ const QString& ShortcutsModel::actionDescription(const std::string& actionCode)
 
 const QString& ShortcutsModel::actionText(const std::string& actionCode) const
 {
-    return actionDescription(actionCode) == "" ? actionTitle(actionCode) : actionDescription(actionCode);
+    const QString description = actionDescription(actionCode);
+    return description.isEmpty() ? actionTitle(actionCode) : description;
```

Comment:
This produces a compiler warning: you are returning a reference to `description`, which is a temporary object and will be deleted at the bottom of this method. So the reference ends up pointing to a deleted object. 

Correct would be:
```suggestion
    const QString& description = actionDescription(actionCode);
    return description.isEmpty() ? actionTitle(actionCode) : description;
```
This does work, because `actionDescription` also returns a reference, so you basically forward that reference this way. 

Also, this method is still not quite efficient: both `actionTitle` and `actionDescription` need to look up the action in the actions register. It would be better to look up the action once, then look at its title/description members directly.

Anyway, this method will be overhauled in my next PR, so this is just for information.

Reply:
I see... Thank you for letting me know of this issue :)